meta/system.py

```diff
@@ -715,6 +715,7 @@ def recog(
             parallelize=parallelize_conversion,
             **lattice_to_ctm_kwargs,
         )
+        lat2ctm.add_alias("%s%s_conv_to_ctm" % (prefix, name))
```

Comment:
We have already 
```
rec.add_alias("%srecog_%s" % (prefix, name))
```
and 
```
self.jobs[corpus]["lat2ctm_%s" % name] = ...
```
so I guess the most consistent formatting would be
```suggestion
        lat2ctm.add_alias("%slat2ctm_%s" % (prefix, name))
```